central/src/gitlab.rs

```diff
@@ -24,6 +24,28 @@ pub struct GitLabProjectAccessTokenProvider {
     client: reqwest::Client,
 }
 
+/// Derive the bridgehead configuration repository from the beam id of the requester.
+/// Example return value: "bridgehead-configurations/bridgehead-config-berlin"
+fn derive_bridgehead_config_repo_from_beam_id(requester: &AppId) -> Result<String, String> {
+    let mut parts = requester.as_ref().splitn(3, '.');
+    let (_app, proxy, broker) = (parts.next().unwrap(), parts.next().unwrap(), parts.next().unwrap());
+
+    let group = match broker {
+        "broker.ccp-it.dktk.dkfz.de" => "bridgehead-configurations", // ccp
+        "broker.bbmri.sample.de" => "bbmri-bridgehead-configs", // bbmri
+        "broker.bbmri.de" => "bbmri-bridgehead-configs", // gbn
+        "test-no-real-data.broker.samply.de" => "bridgehead-configurations", // cce, itcc, kr
+        "broker.hector.dkfz.de" => "dhki", // dhki
+        _ => return Err(format!("Bridgehead configuration repository not known for beam id {requester}")),
+    };
```

Comment:
Not sure what you discussed with Torben and Martin but from my side I think I would have made this a config option instead of deriving it from the broker id but that seems fine as well

Reply:
I assume you imply there is one central secret sync per broker? It would make sense then, yes.

Reply:
Yeah there pretty much has to be because we have to actively pull tasks from the broker and we really ever poll from one broker. But leave it like that it will be fine